federationapi/storage/postgres/queue_edus_table.go

```diff
@@ -61,6 +63,12 @@ const selectQueueEDUCountSQL = "" +
 const selectQueueServerNamesSQL = "" +
 	"SELECT DISTINCT server_name FROM federationsender_queue_edus"
 
+const selectExpiredEDUsSQL = "" +
+	"SELECT DISTINCT json_nid FROM federationsender_queue_edus WHERE expires_at IS NOT NULL AND expires_at <= $1"
+
+const deleteExpiredEDUsSQL = "" +
+	"DELETE FROM federationsender_queue_edus WHERE expires_at IS NOT NULL AND expires_at <= $1"
```

Comment:
Presumably then we'll never get rid of EDUs stored prior to this PR as the expires value will be NULL? It might be worth doing something like `DEFAULT now()` or something?